Sources/XCTest/XCTimeUtilities.swift

```diff
@@ -13,24 +13,24 @@
 
 #if os(Linux) || os(FreeBSD)
     import Glibc
+    import Foundation
 #else
     import Darwin
+    import SwiftFoundation
```

Comment:
Perhaps we should borrow a page from swift-package-manager and [extract these checks into a single location](https://github.com/apple/swift-package-manager/blob/eb8c15ce30cccd5c53a5d4d7367a94bc3df0d084/Sources/libc/libc.swift#L11-L15).

Something to consider for another pull request! :point_up: